dataactcore/models/validationModels.py

```diff
@@ -28,8 +28,8 @@ class FileColumn(Base):
     __tablename__ = "file_columns"
 
     file_column_id = Column(Integer, primary_key=True)
-    file_id = Column(Integer, ForeignKey("file_type_validation.file_id"), nullable=True)
-    file = relationship("FileTypeValidation", uselist=False)
+    file_id = Column(Integer, ForeignKey("file_type.file_type_id", name="fk_file_column_file_type"), nullable=True)
```

Comment:
Curious: it looks like we don't consistently name our FKs (often using the default sqlalchemy generates) -- is this a step towards standardizing on explicit FK names?